bHY6yh-de2065/cod-yyyiou_bHY6yh-de2065.xml

```diff
@@ -0,0 +1,193 @@
+          <lb ed="#C"/>pereat qui ablectus est</quote>
+        </p>
+        <p xml:id="bHY6yh-de2065-d1e267">
+          ¶ <ref xml:id="bHY6yh-de2065-Rd1e287">Tullius in o<lb ed="#C"/>ratione 
+          pro quinto ligario</ref> <quote xml:id="bHY6yh-de2065-Qd1e271">homies ad de<lb ed="#C"/>os
+          nulla re propitius accedunt quam salu<lb ed="#C"/>tem 
+          hominibus dando </quote>
+        </p>
+        <p xml:id="bHY6yh-de2065-d1e279">
+          ¶ <ref xml:id="bHY6yh-de2065-Rd1e301">Ioannes in apocalipsi capitulo 3
+          <lb ed="#C"/></ref> <quote xml:id="bHY6yh-de2065-Qd1e283">Ait dominus Sto ad ostium et pulso siquis au<lb ed="#C"/>dierit 
+          uocem meam et aperuerit ianu<lb ed="#C"/>am 
+          introibo ad illum et cenabo cum illo 
+          <lb ed="#C"/>et ille mecum</quote>  
+        </p> 
+        <p xml:id="bHY6yh-de2065-d1e293">
+          ¶ <ref xml:id="bHY6yh-de2065-Rd1e315">Idem in euangelo capitulo III</ref> <quote xml:id="bHY6yh-de2065-Qd1e295">Sic 
+          <lb ed="#C"/>deus duxit mundum ut filium suum
+          <lb ed="#C"/>vnigenitum daret ut omnis qui credit
+          <lb ed="#C"/>in illum non pereat habeat sed uitam eternam
+          <lb ed="#C"/>non enim misit deus filium suum in mund<lb ed="#C"/>um
+          ut iudicet mundum sed ut saluetur 
+          <lb ed="#C"/>mundus per ipsum </quote>
+        </p>
+        <p xml:id="bHY6yh-de2065-d1e309">
+          ¶ <ref xml:id="bHY6yh-de2065-Rd1e333">Idem XV </ref> <quote xml:id="bHY6yh-de2065-Qd1e311">ait deus
+          <lb ed="#C"/>si praecepta mea seruaueritis manebitis in di<lb ed="#C"/>lectione 
+          mea. </quote>
+        </p>
+        <p xml:id="bHY6yh-de2065-d1e325">
+           ¶ <ref xml:id="bHY6yh-de2065-Rd1e353">petrus apostolus in epistola II capitulo 
+           <lb ed="#C"/>III et fi</ref> <quote xml:id="bHY6yh-de2065-Qd1e329">deus patienter agit propter nos nolens 
+           <lb ed="#C"/>aliquem perire sed omnes ad penitentiam reuerti</quote>
+        </p>
+        <p xml:id="bHY6yh-de2065-d1e337">
+          ¶ <ref xml:id="bHY6yh-de2065-Rd1e367">Paulus ad romanos capitulo VIII</ref> <quote xml:id="bHY6yh-de2065-Qd1e341">qui cumque spiritu dei 
+          <lb ed="#C"/>agiuntur hi filii sunt dei</quote>
+        </p>  
+        <p xml:id="bHY6yh-de2065-d1e348">
+          ¶ <ref xml:id="bHY6yh-de2065-Rd1e380">Cassiodorus 
+          <lb ed="#C"/>epistolarum libro VIII capitulo XXVI</ref> <quote xml:id="bHY6yh-de2065-Qd1e352">Quos superna<cb ed="#C" n="b"/><lb ed="#C"/>ptegunt
+          felices aduersarios hre non pomient</quote>
+        </p>
+        <p xml:id="bHY6yh-de2065-d1e358">
+          <!-- next verse is not present in text-->
+          <ref xml:id="bHY6yh-de2065-Rd1e392">Prouerb.vulgar.</ref> <quote xml:id="bHY6yh-de2065-Qd1e362">A chi dio vol bien: dor
+          mando ieuen. A chin dio vol male: elge tole el
+          seno. A chin dio vol male le piegore ge da
+          de dente </quote>
```

Comment:
If this text isn't present in your text we need to delete the content and write [omitted] like so:

```suggestion
        <p xml:id="bHY6yh-de2065-d1e358">
          [omitted]
        </p>
```